modules/serverpod_auth/serverpod_auth_server/lib/src/business/firebase_auth.dart

```diff
@@ -1,25 +1,28 @@
-import 'package:firebase_admin/firebase_admin.dart';
+import 'dart:convert';
+import 'dart:io';
+
 import 'package:serverpod_auth_server/serverpod_auth_server.dart';
+import 'package:serverpod_auth_server/src/firebase/firebase_admin.dart';
 
 /// Convenience methods for handling authentication with Firebase.
 class FirebaseAuth {
-  static App? _app;
+  static Auth? _auth;
 
   /// Returns the Firebase app.
-  static App get app {
-    if (_app != null) {
-      return _app!;
+  static Future<Auth> get auth async {
+    if (_auth != null) {
+      return _auth!;
     }
 
-    var cert = FirebaseAdmin.instance.certFromPath(
```

Comment:
Suggestion: Please preserve the "cert" naming here if that is what the file we are reading from disk is.

The variable name `json` doesn't really communicate what we are loading here.